packages/schemas/src/db-entries/setting.ts

```diff
@@ -0,0 +1,74 @@
+// THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+
+import { z } from 'zod';
+
+import {
+  SigningKeyManagementPayload,
+  signingKeyManagementPayloadGuard,
+  ApiAuthorizationSettingsPayload,
+  apiAuthorizationSettingsPayloadGuard,
+  LoginAndLogoutPayload,
+  loginAndLogoutPayloadGuard,
+  GeneratedSchema,
+  Guard,
+} from '../foundations';
+import { TenantRegionType } from './custom-types';
+
+export type SettingDBEntry = {
+  tenantId: string;
+  tenantName: string;
+  tenantRegion?: TenantRegionType;
```

Comment:
no need for this? we don't have the concept of "tenant" yet.

Reply:
I think the setting is actually a tenant-level configuration? in V1 stage, tenant-level setting means global setting?

Reply:
i mean we do not have the concept of "tenant" in v1